### ISSUE TITLE
[BUG] almalinux 9 , installation, step1-3 passed check,but step 4 :  " Please fix errors before continuing to the next step".

### ISSUE THREAD

**Issue by catigerose** [FreshRSS/FreshRSS#4933]
**Describe the bug**
almalinux 9(Installed Environment Groups:   Minimal Install), the step 1-3 of Installation all passed check，but step 4 say： " Please fix errors before continuing to the next step".

**To Reproduce**
1. I  buy a vps in Racknerd, choose OS :  almalinux 9 (Minimal Install); 
2. install apache: 
   sudo dnf install httpd httpd-tools -y
    sudo systemctl start httpd 
   sudo systemctl enable httpd
   
    version：Apache/2.4.53 (AlmaLinux)

3. install php and extensions：
 sudo dnf install -y php php-curl php-gmp php-intl php-mbstring php-xml php-zip php-ctype php-dom php-fileinfo php-iconv php-json php-opcache php-openssl php-phar php-session php-simplexml php-xmlreader php-xmlwriter php-xml php-tokenizer php-zlib php-pdo_sqlite php-pdo_mysql php-pdo_pgsql

 PHP-FPM：   sudo systemctl start php-fpm && sudo systemctl enable php-fpm 

    version ：PHP 8.0.20 (cli)

4. reboot  linux

5. install FreshRSS:
      sudo dnf install -y git  
   cd /usr/share/ 
 git clone https://github.com/FreshRSS/FreshRSS.git   
 cd FreshRSS
chown -R apache:apache  .
sudo chmod -R g+r .
sudo chmod -R g+w ./data/
sudo chmod -R g+w .
ln -s /usr/share/FreshRSS/p /var/www/html/

6. visit：http:// myip/p  ;  start installation
  step 1- 3 all passed check;   
step 4: imput user name and password ; then submit ;  but failed with warnings:" Please fix errors before continuing to the next step".
（in step 2   choose SQLite or install  MySQL ,results are same  ） 

PS: I do the same steps above in   almalinux 8  (PHP 7.2.24   ;   Apache/2.4.37 (AlmaLinux)),  FreshRSS is  installed and  works fine.    

<img width="487" alt="image" src="https://user-images.githubusercontent.com/67047102/206684968-0ccc3ec1-1114-4aff-9e5b-40491b90cfd5.png">
<img width="376" alt="image" src="https://user-images.githubusercontent.com/67047102/206685040-f7aea5a3-03c4-4772-b3e5-b6020669f1b7.png">
<img width="420" alt="image" src="https://user-images.githubusercontent.com/67047102/206685140-83b5c300-9d3d-4d63-8859-0333942781dd.png">

**Comment by Alkarex** [comment]
Did you find a solution to the problem?

**Comment by catigerose** [comment]
No，I am layman in programming and Using almalinux 8 now.

Vultr's(another VPS) almalinux 9 works fine.(vultr ：yum grouplist ：Server
Racknerd ：yumg grouplist ：  minimal install.）
I tried groupinstal server with GUI or other packages in racknerd.it didn't work.

I think the problem may not be important，or it's not FresRSS' problem. So I closed this issue.

FreshRSS is really excellent! I love it very much . Thanks for your efforts!
